### PR TITLE
[ADD] travis configuration testing docklint

### PR DESCRIPTION
**Pull request by osvalr**
This PR respond to the #39 issue, Dockerfiles are tested but I DO need a help for more flexible travis configuration

### CODE REVIEW

**Comment by osvalr** [comment]
@moylop260 feel free to make fixes or comments as well

**Comment by osvalr** [comment]
@moylop260 I was thinking that it would be a great idea to include script section ( currently raw shell script) as another script file into MQT, but let me know that you think about

**Comment by osvalr** [comment]
help with this..
```bash
find: `/home/travis/build/vauxoo/docker-odoo-image': No such file or directory
```

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    WORKINGDIR="${HOME}/build/vauxoo/docker-odoo-image"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - npm install -g validate-dockerfile
+
+script:
+  - for dfile in `find ${WORKINGDIR} -name 'Dockerfile' -not -path "./.git/*"`; do echo "[ CHECKING $dfile ]"; docklint $dfile; if [ $? -ne 0 ]; then return 1; fi; echo ""; done
```
Comment (moylop260) [inline .travis.yml]: s/WORKINGDIR/TRAVIS_BUILD_DIR/g

Comment (osvalr) [inline .travis.yml]: thank you

**Pull request merged**